contents/handbook/growth/sales/customer-onboarding.md

```diff
@@ -1,37 +1,116 @@
+1. Are likely to contract above $20k with us.
+   (Ideally we qualify this by giving indicative pricing in the demo)
+2. Are likely to enter into an annual contract.
+   (This is quite a high-effort process for people just going month to month)
+3. Are ready to get hands-on with PostHog and will make a decision in weeks, not months.
+
+### Expectations of the customer
+
+We'll need them to be able to demo their product to us, as well as attend two or more zoom calls where we scope out the data and help them get set up.
+
+Ideally we will also have them in Slack Connect channel so that we can provide responsive support and expose them to the wider PostHog team.
+
+## Day 0 - Kicking things off
+
+At the start of the evaluation, we want to review their product to understand and advise on the best approach to tracking, as well as address any privacy concerns associated with session recordings.
+
+By the end of the call we should have a plan for event capture/opt-out capture and an agreed timeline to get that set up.
+
+### Prerequisites
+
+The customer should come prepared to demo their product to us, where we can help figure out the key tracking events needed for the evaluation to be successful.
+
+If they don't already know about AARRR we should share our [AARRR blog post](https://posthog.com/blog/aarrr-pirate-funnel) and [Tracking Plan](https://docs.google.com/spreadsheets/d/12uV5aKAhU_wygUQl3YXZU2J_QN_AZi4nPFj-9WIKhlY/edit#gid=0) and ask them to review it before the call.
+
+### Structure
+
+1. Review goals and structure of this session 
+2. Review key concepts:
+   * Acquisition
+   * Activation
+   * User Identification
+   * Cohorts
+   * Groups
+   * Privacy / opt out capture
+3. Have customer demo their app to you, focusing on where the above information is captured
+   * During the demo agree where Acquisition/Activation/Identification take place
+   * Get the CSS selectors and pages of any items to opt-out of capture
+   * Agree any additional properties that need to be captured
+4. Recap and agree the tracking and other implementation details
+5. Agree the timeline to have tracking implemented and set up the following call (ideally 3 days after capture is implemented)
+
+### Deliverables
+
+1. A partially filled-in tracking plan detailing Activation and Acquisitoon
+2. A code snippet showing them how to implement tracking for their product (including Identification and Groups if applicable)
+3. Elements and pages to add opt-out-capture to
+
+## Day 3 - Using PostHog
+
+The aim of this call is to get the customer familiar with navigating PostHog as well as:
+
+* Defining Actions
+* Defining Cohorts
+* Creating Insights
+* Creating Dashboards
+* Finding Recordings
+
+As much as possible the customer should be sharing their screen and driving the session, by teaching them to fish they become comfortable and self-sufficient with PostHog.
+
+### Prerequisites
```

Comment:
I remember we had a couple of prerequisites for us as well right? Check if the data is coming in correctly and something else?

Reply:
As far as I remember it's just sharing the data capture info after the last call?